### DISCUSSION TITLE
Register Github Education using regular email

### DISCUSSION THREAD

**Discussion by kevinalfito69** [community/community#141674] [GitHub Education] [answered]
### Hi everyone,

Hi everyone , I want to register to github education I have an official id from my campus but my campus does not provide a campus email is there another solution so that I can register to github education without a campus email?

**Comment by BrunoL0M** [comment]
no creo que se puede por que si mal no recuerdo primero tienes que enlazar tu correo universitario con tu cuenta de github principal hay una seccion en la parte de configuracion en emails

**Reply by BrunoL0M** [reply, marked as answer]
I don't think you can because if I remember correctly you first have to link your university email with your main GitHub account there is a section in the configuration part in emails

**Comment by henry-AY** [comment]
Hey kevinalfito69,

If I remember correctly, you can only access the benefits if you verify your student email, and then upload an ID, transcript, etc...

You can try to submit a ticket via this [link](https://support.github.com/contact/education), which may let you work around this issue.

If your ticket gets accepted, the waiting time to receive the rewards may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

* Congrats! Your request for @kevinalfito69 was approved and your benefits are now available.
